src/switch_rtp.c

```diff
@@ -7673,7 +7758,7 @@ static void check_timeout(switch_rtp_t *rtp_session)
 		elapsed = (now - rtp_session->last_media) / 1000;
 	}
 
-	switch_log_printf(SWITCH_CHANNEL_SESSION_LOG(rtp_session->session), SWITCH_LOG_DEBUG10,
+	switch_log_printf(SWITCH_CHANNEL_SESSION_LOG(rtp_session->session), SWITCH_LOG_DEBUG1,
```

Comment:
Checking if this is just a typo or intentional. No issue for me if you lower the debug level

Reply:
It's been lowered to DEBUG1, I think this is fine (note this won't get printed at all if there's no issues), cause media timeout check is called only when issues are likely.

Reply:
This won't be printed cause DEBUG1 is too low for both prod and dev - these levels are not printed.